code.gs

```diff
@@ -766,7 +766,7 @@ DataCollector.prototype.getProp = function (key) {
  * @param {?string} value - The value of the property.
  */
 DataCollector.prototype.setProp = function (key, value) {
-  this.prop[key] = (typeof value !== 'undefined' && value !== '' ? value : null);
+  this.prop[key] = (value ? value : null);
```

Comment:
My semistandard test throws an error here:

```
file: 'code.gs'
severity: 'Error'
message: 'Unnecessary use of conditional expression for default assignment. (no-unneeded-ternary)'
at: '769,29'
source: 'semistandard'
code: 'no-unneeded-ternary'
```

It looks like in the semistandard spec the `no-unneeded-ternary` rule is configured with `"defaultAssignment": false` (see [ESLint documentation](https://eslint.org/docs/rules/no-unneeded-ternary)).  
Adding `/* eslint no-unneeded-ternary: ["error", { "defaultAssignment": false }] */` at the top of the file fixes this problem